helmfile/charts/init-harbor/values.yaml

```diff
@@ -3,7 +3,8 @@ endpoint: harbor-harbor-core/api/v2.0
 password: "Harbor12345"
 
 oidc:
-  scope: "openid,email,profile,offline_access"
+  scope: "openid,email,profile,offline_access,groups"
   clientSecret: ""
   endpoint: ""
   verifyCert: true
+  groupClaimName: {{ .Values.harbor.groupClaimName }}
```

Comment:
This should be a default value and `{{ .Values.harbor.groupClaimName }}` should be moved to https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/init-harbor.yaml.gotmpl

Reply:
done

Reply:
I still don't see any change in the file `helmfile/values/init-harbor.yaml.gotmpl`

Reply:
working on it - ending up with a detached head so fixing that

Reply:
and done now